release/tag/goreleaser.yaml

```diff
@@ -45,3 +56,5 @@ release:
   github:
     owner: GoogleContainerTools
     name: kpt
+  draft: true
```

Comment:
+1 for these changes, suggest appending a step to the releasing README that includes testing/changing these flags. Otherwise the release will not be public to non-maintainers.

Reply:
Updated the README doc.